docker-compose-rule-core/src/main/java/com/palantir/docker/compose/connection/waiting/ClusterHealthCheck.java

```diff
@@ -37,5 +43,26 @@ static ClusterHealthCheck serviceHealthCheck(String containerName, HealthCheck<C
         };
     }
 
-    SuccessOrFailure isClusterHealthy(Cluster cluster);
+    static ClusterHealthCheck nativeHealthChecks() {
```

Comment:
Would it be feasible to put some logging in here?  It's pretty frustrating to debug an existing healthcheck, so it would be cool if the new ones are better ;)